js_modules/dagit/packages/core/src/launchpad/LaunchpadRoot.tsx

```diff
@@ -71,6 +71,16 @@ export const JobLaunchpad: React.FC<{repoAddress: RepoAddress}> = (props) => {
     return <Redirect to={`/locations/${repoPath}/pipeline_or_job/${pipelinePath}`} />;
   }
 
+  if (pipelinePath.includes('__ASSET_JOB')) {
```

Comment:
Hmm we have a constant for this value - ASSET_JOB_PREFIX I think

Reply:
TIL, thanks!